internal/sirius/add_document.go

```diff
@@ -0,0 +1,58 @@
+package sirius
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+func (c *Client) AddDocument(ctx Context, caseID int, document Document, docType string) (Document, error) {
+	data, err := json.Marshal(struct {
+		CaseId          int    `json:"caseId"`
+		CorrespondentID int    `json:"correspondentId"`
+		Type            string `json:"type"`
+		FileName        string `json:"filename"`
+		SystemType      string `json:"systemType"`
+		Content         string `json:"content"`
+	}{
```

Comment:
it would be nice to name this as a (private) type, I try to keep anonymous ones to tests mostly